Editor/Resources/EditorWindow/Pages/LandingPage.uss

```diff
@@ -0,0 +1,72 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+@import "./common.uss";
+
+:root {
+    --color-card-border: #0d0d0d;
+    --color-card-background: #2a2a2a;
+}
+
+.cards {
+    flex-direction: row;
+}
+
+.card {
+    padding: 12px;
+    border-width: 2px;
+    border-color: var(--color-card-border);
+    background-color: var(--color-card-background);
+    flex-basis: 50%;
```

Comment:
The defautl style for this includes `flex: 0 1 auto`, so the cards will not grow to full the space unless either `flex-grow: 1` or `flex-basis: 50%` is specified. The former being preferred I think instead of an arbitrary basis and using `flex-shrink` to get it down to size

Reply:
infact, size the width is driven by the content, if we dont set flex-basis, the cards can be different widths

Reply:
Ah I see, I missed that `flex-basis: auto` means it starts with the content size; however, if we specify anything other than `auto`, it will scale equally.

Consider updating it to `0` as `50%` implies there's a tight coupling with there only being two cards when there isn't because the result is the same